File: cpp/Ram.cpp
#include <iostream>
using namespace std;

class Ram
{
    private:
        string capacity = "";
        int price = 0;

    public:
        Ram(){}

        Ram(string capacity, int price)
        {
            this->capacity = capacity;
            this->price = price;
        }

        void setCapacity(string capacity)
        {
            this->capacity = capacity;
        }

        string getCapacity()
        {
            return this->capacity;
        }

        void setPrice(int price)
        {
            this->price = price;
        }

        int getPrice()
        {
            return this->price;
        }

        void showRam()
        {
            cout << "RAM Capacity    : " << this->getCapacity() << endl;
            cout << "RAM Price       : " << this->getPrice() << endl;
        }

        ~Ram(){}
};

File: python/Ram.py
class Ram:
    capacity = ""
    price = 0

    def __init__(self, capacity, price):
        self.capacity = capacity
        self.price = price

    def setCapacity(self, capacity):
        self.capacity = capacity

    def getCapacity(self):
        return self.capacity

    def setPrice(self, price):
        self.price = price

    def getPrice(self):
        return self.price

    def showRam(self):
        print("RAM Capacity    : " + self.getCapacity())
        print("RAM Price       : " + str(self.getPrice()))